sdk/core/azure-core-amqp/pom.xml

```diff
@@ -154,7 +154,7 @@
             <bannedDependencies>
               <includes>
                 <include>com.microsoft.azure:qpid-proton-j-extensions:[1.2.4]</include> <!-- {x-include-update;com.microsoft.azure:qpid-proton-j-extensions;external_dependency} -->
-                <include>org.apache.qpid:proton-j:[0.33.8]</include> <!-- {x-include-update;org.apache.qpid:proton-j;external_dependency} -->
+                <include>org.apache.qpid:proton-j:[0.34.1]</include> <!-- {x-include-update;org.apache.qpid:proton-j;external_dependency} -->
```

Comment:
It seems this proton-j-extensions also depends on proton-j version, but it's a library not in this mono repro, so we need to track down who owns it and how to release it.

https://github.com/Azure/qpid-proton-j-extensions/blob/1c372719aa1403f4ed2b18468462e07276cb128b/pom.xml#L48

Reply:
Okay direct owners of this library is JavaSDK team now. I can see Connie, me and v-team in the owner list, we can sync internally.